### PR TITLE
fix: add ability to set the log level via environment variable

### PR DESCRIPTION
**Pull request by iloveitaly**
As discussed in https://github.com/ActivityWatch/aw-client/pull/57/files﻿ this PR adds the ability to set the log level of all AW components via a ENV variable.

### CODE REVIEW

**Comment by iloveitaly** [comment]
Friendly reminder here @ErikBjare

File: aw_core/log.py [hunk 1]
```diff
@@ -32,6 +32,11 @@ def setup_logging(
     root_logger.setLevel(logging.DEBUG if verbose else logging.INFO)
     root_logger.handlers = []
 
+    # run with LOG_LEVEL=DEBUG to customize log level across all AW components
+    log_level = os.environ.get('LOG_LEVEL')
+    if log_level:
+        root_logger.setLevel(logging.__getattribute__(log_level.upper()))
```
Comment (ErikBjare) [inline aw_core/log.py:38]:
```suggestion
        if hasattr(log_level, log_level.upper()):
            root_logger.setLevel(getattr(logging, log_level.upper()))
        else:
            root_logger.warning(f'No logging level called {log_level} (as specified in env var)')
```

**Review by ErikBjare — COMMENTED** [review]
Sorry, got a bit distracted by the underscore in the env var (LOGLEVEL vs LOG_LEVEL) but it doesn't really matter.

I have one suggestion though. If you're okay with it and it looks good (haven't tested it) I'll apply it and merge :)

File: aw_core/log.py (hunk 1 above)
Comment (iloveitaly) [reply aw_core/log.py:38]: Ah, wasn't aware of those functions! Thanks.

**Comment by ErikBjare** [comment]
Slowpoke here, finally getting around to merging this. Thanks!

**Pull request merged**